packages/terra-application-layout/src/utils/propTypes.js

```diff
@@ -42,6 +42,7 @@ const utilityConfigPropType = PropTypes.shape({
   title: PropTypes.string,
   accessory: PropTypes.element,
   onChange: PropTypes.func.isRequired,
+  onBlur: PropTypes.func,
```

Comment:
It does not appear that the documentation has been updated to account for this new API.